lib/src/editor/ui/editor_tabs.dart

```diff
@@ -197,12 +130,91 @@ class _CoreContentTabsState extends State<CoreContentTabs> {
               },
             ),
           ),
+          const SizedBox(width: 8),
         ],
       ),
     );
   }
 }
 
+Widget defaultControls(context) {
+  return Consumer<DevicePreviewProvider>(builder: (context, model, child) {
+    return Wrap(
+      children: [
+        TextButton(
+          onPressed: context.read<ZoomProvider>().zoomIn,
+          style: TextButton.styleFrom(
+            splashFactory: InkRipple.splashFactory,
+            shape:
+                RoundedRectangleBorder(borderRadius: BorderRadius.circular(90)),
```

Comment:
Not a show stopper... but are there any plans to externalize values such as radiuses, paddings, widths, icon sizes, or even creating widgets for buttons. This would keep future changes minimum and scoped.

Reply:
Not sure how in the weeds we want to get with managing this OS package, might be a discussion worth asking @devbrandonlong about